### DISCUSSION TITLE
Users can see Other Users Task under same Project

### DISCUSSION THREAD

**Discussion by ghost** [Leantime/leantime#1123] [Project Management] [answered]
I have created 1 project & assigned some tasks to some users with editor access to let them able to modify Task Progress.

All users can see each other task & is able to modify it.

How to restrict them from viewing other users task?

**Comment by marcelfolaron** [comment, marked as answer]
Hi
Thanks for reaching out.

If I understand you correctly you would like to have a project where users
can only see their individually assigned tasks?
This is currently not possible within Leantime however you could create
separate projects for each user to "hide" the tasks from each other.

To be honest this is likely not something we'd be building since our entire
premise is built around collaboration within one project. There may be an
option to build this as a plugin once we are done writing the plugin system
(upcoming version 2.3)
